File: problems/geeksforgeeks/medium/kth_smallest.go
package main

import (
	"fmt"
	"sort"
)

func kthSmallest(arr []int, k int) int {
	sort.Ints(arr)

	return arr[k-1]
}

func main() {
	arr := []int{7, 10, 4, 20, 15}
	k := 4

	res := kthSmallest(arr, k)
	fmt.Printf("The %vth smallest is %v\n", k, res)
}

File: problems/geeksforgeeks/medium/kth_smallest.rs
fn kth_smallest(arr: &mut [i32], k: usize) -> i32 {
    arr.sort();
    arr[k - 1]
}
fn main() {
    let mut arr = [7, 10, 4, 20, 15];
    let k = 4;

    let res = kth_smallest(&mut arr, k);
    println!("The {}th smallest is {}", k, res);
}
